### ISSUE TITLE
User information security

### ISSUE THREAD

**Issue by DolphinLHT** [henrychu04/CS419-Computer-Security-Project#4]
The user's information should be encrypted in some way potentially.

**Comment by henrychu04** [comment]
All the important things, such as messages and passwords, are encrypted. The unencrypted information that is stored on the server are usernames and a third party can access the usernames only if they gain access to the server. As of now, since both the server and client are run on the same computer, all the usernames are accessible to the user. But if the server is run on a cloud service, then the usernames will not be accessible.

To put it simply, all the important things are encrypted and there is no need to encrypt the usernames, which is the only information that is not encrypted.

**Comment by symxmyz233** [comment]
**It is not a bug.**
